content/agent/_index.md

```diff
@@ -3,103 +3,137 @@ title: Agent
 kind: documentation
 description: Install & configure the Agent to collect data
 aliases:
-    - /guides/basic_agent_usage/
+    - /agent/faq/agent-status-and-information
+    - /agent/faq/start-stop-restart-the-datadog-agent
+    - /agent/faq/send-logs-and-configs-to-datadog-via-flare-command
+    - /agent/faq/how-to-get-more-logging-from-the-agent
 ---
 
-{{< partial name="platforms/platforms.html" >}}
+<div class="alert alert-info">
+    Agent v6 is now available, <a href="https://github.com/DataDog/datadog-agent/blob/master/docs/agent/upgrade.md">upgrade to the newest version </a> to benefit from all new functionalities 
+</div>
 
 ## What is the Agent?
 
 The Datadog Agent is a piece of software that runs on your hosts. Its job is to faithfully collect events and metrics and bring them to Datadog on
-your behalf so that you can do something useful with your monitoring and performance data.
+your behalf so that you can do something useful with your monitoring and performance data. The Datadog Agent is open-sourced. You may view the source code on GitHub for [Agent v5](https://github.com/DataDog/dd-agent) and [Agent v6](https://github.com/DataDog/datadog-agent). To see all changes between Agent v5 and v6, [consult our dedicated changes documentation](https://github.com/DataDog/datadog-agent/blob/master/docs/agent/changes.md).
 
-[The source code for the Datadog Agent](https://github.com/DataDog/dd-agent).
+{{< partial name="platforms/platforms.html" >}}
 
-For information on running the Agent through a proxy, see [the dedicated documentation Datadog Agent and proxy ](/agent/proxy); for which IP ranges to allow, see [the complete list of IP's and ports](https://github.com/DataDog/dd-agent/wiki/Network-Traffic-and-Proxy-Configuration#open-ports).
+## Start/Stop/Restart the agent 
```

Comment:
Can we change this to something like "Operating the Agent service"?

Reply:
Is there a distinction?